.jenkins/pipelines/nightly.Jenkinsfile

```diff
@@ -94,18 +94,8 @@ pipeline {
                     }
                 }
                 excludes {
-                    // Skip builds with Ubuntu 18.04 and v3
-                    exclude {
-                        axis {
-                            name 'OS_VERSION'
-                            values '18.04'
-                        }
-                        axis {
-                            name 'VM_GENERATION'
-                            values 'v3'
-                        }
-                    }
                     // Skip builds with Ubuntu 20.04 and v2
```

Comment:
To confirm, are we still skipping 20.04 here?

Reply:
Yes this comment is still applicable:
* We are skipping Ubuntu 20.04 on Coffee Lake for 'DotNet', 'DotNet-P1', 'Azure-SDK', 'PyTorch', 'OpenMP-Testsuite'
* But now running Ubuntu 20.04 on Coffee Lake for 'Unit' and 'Solutions' tests